src/index.html

```diff
@@ -12,7 +24,53 @@
       href="./styles/index.scss"
```

Comment:
According to the code style rules, SCSS files should be imported without file extension. So you should use './styles/index' instead of './styles/index.scss'.